README.md

```diff
@@ -13,6 +13,19 @@ Add them to your project at:
 - `ios/Runner/GoogleService-Info.plist`
 - `android/app/google-services.json`
 
+## Cloud Functions 
+
+We are using Cloud Functions for Firebase to automatically run backend code in response to events triggered by Firebase features, such as: 
+- Initial Sign In
+- Changes to Firestore that should send an FCM
+
+The relevant code is all in `functions/`
+
+To deploy: 
+```
+firebase deploy --only functions
```

Comment:
Does it take the [firebase cli](https://firebase.google.com/docs/cli) installed to run this?

Reply:
Thanks @tourdownunder!  Yes, you do indeed need the firebase cli - as you kindly indicate, the README is missing important info on how to use the cloud functions.  I'll add this to the README before I merge.  Thanks!